testsuite/simulation/modelica/commonSubExp/cseTestCall1.mos

```diff
@@ -103,7 +103,7 @@ simulate(CSE.TestCall1); getErrorString();
 // ========================================
 // 1: $cse1 - log($cse2) - {}
 // 2: $cse2 - log(D) - {}
-// 3: D - log(A) - {}
+// 3: D - log(4.0 * C) - {}
```

Comment:
I believe it's would be good, if at least one test is left for `removeEqualFunctionCalls` e.g. using
with `preOptModules+=removeEqualFunctionCalls`!